test/e2e/e2e_test.go

```diff
@@ -347,6 +347,63 @@ func TestE2EFlow(t *testing.T) {
 
 	})
 
+	t.Run("Try to delete namespaced scoped resources of users, and expect recreation", func(t *testing.T) {
+		userSignup, _ := NewSignupRequest(t, awaitilities).
+			Username("wonderwoman").
+			Email("wonderwoman@redhat.com").
+			ManuallyApprove().
+			EnsureMUR().
+			TargetCluster(memberAwait).
+			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
+			Execute().Resources()
+		devNs := corev1.Namespace{}
+		err := memberAwait.Client.Get(context.TODO(), types.NamespacedName{Name: "wonderwoman-dev"}, &devNs)
+		require.NoError(t, err)
+
+		stageNs := corev1.Namespace{}
```

Comment:
You are not testing the stage namespace. Let's parameterize the actual tests so they test the deleted role/rolebindgins in the provided namespace and run them against both namespaces.

Reply:
done in 74a0068